### PR TITLE
Analysis/simulations

### LINKED ISSUE #3: Include the 2-gene simulations
Include all notebooks and scripts necessary to reproduce preprint results.

### PR DESCRIPTION
**Pull request by zoepiran**
Includes:
1. 2-gene simulation scripts and analysis
2. zebrafish scripts
3. Tedsim results analysis and scripts (@michalk8 pls validate the latter + update a README for running the Tedsim scripts)

closes #3 #4

### CODE REVIEW

File: analysis/simulations/forrow_2gene/utils.py [hunk 1]
```diff
@@ -0,0 +1,933 @@
...
+    plt.tight_layout()
+    plt.show()
+
+
+def plot_trajectory(
+    sim_dict,
+    flow_type,
+    thr=1e-8,
+    alpha_scale=1,
+    # TODO(zoepiran): unused after deleting unused variables
```
[the remark this answers is not in the document]
Comment (zoepiran) [reply analysis/simulations/forrow_2gene/utils.py:846]: done

File: analysis/simulations/forrow_2gene/README.rst [hunk 2]
```diff
@@ -0,0 +1,14 @@
+2-gene analysis
+===============
+
```
[the remark this answers is not in the document]
Comment (zoepiran) [reply analysis/simulations/forrow_2gene/README.rst:3]: done

**Comment by zoepiran** [comment]
All requests addressed :)

**Comment by michalk8** [comment]
> you need to supply config.yml

I added the missing config files, thanks a lot @zoepiran ! Merging this.

**Pull request merged**